svc-aggregation/README.md

```diff
@@ -638,8 +657,8 @@ curl -i GET \
    "HostName":"10.24.0.4",
    "UserName":"admin",
    "Links":{
-      "ConnectionMethod": {
-         "@odata.id": "/redfish/v1/AggregationService/ConnectionMethods/d172e66c-b4a8-437c-981b-1c07ddfeacaa"
+      "Oem":{
+         "PluginID":"GRF"
```

Comment:
the response should have the connection method info, not the   plugin id please revert the same

Reply:
Changed.